src/modules/authorization/sagas.ts

```diff
@@ -30,29 +30,16 @@ function* handleFetchAuthorizationsRequest() {
 
   const qs = new URLSearchParams(document.location.search)
 
-  const authorizationServer = qs.get('authorizationServer')
-  if (authorizationServer) {
+  const skipValidations = qs.get('skipValidations')
+  if (skipValidations) {
     try {
       const { parcels } = info
-      const address: string = yield select(getAddress)
-
-      const res: Response = yield call(() => {
```

Comment:
Were is the request to the auth server being made now?

Reply:
https://github.com/decentraland/cli/pull/801 clear now that I've read the conversation on that pr

Reply:
If I'm correct, the is no auth server to get permissions now. Instead, client side validations are skipped to make it easier